modules/react/src/deckgl.js

```diff
@@ -73,6 +71,11 @@ export default class DeckGL extends React.Component {
         })
       );
     this._updateFromProps(this.props);
+
+    // Bind public methods
```

Comment:
Nits:
1. This change seems unrelated.
2. This is a little too terse for my tastes, I would just keep and bind the methods here. deck is complicated and as a rule I think it is worth going for simpler standardized code.